.gitignore

```diff
@@ -1 +1,3 @@
 target/
+.idea
```

Comment:
This is an option but most project I work with has `.idea` added directly in in local gitignore (in this dir intellij stores whole project config). 

I can remove `.idea` if needed.

`zinc/src/test/resources/bin` keeps generated jars/classesdirs